types/svg64/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for svg64 1.1
+// Project: https://github.com/scriptex/svg64
+// Definitions by: Ben Grynhaus <https://github.com/bengry>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+export as namespace svg64;
+
+/*~ If this module has methods, declare them as functions like so.
+ */
+declare function svg64(svgStringOrElement: string | SVGElement): string;
+
+export default svg64;
```

Comment:
this should be probably:
```ts
export = svg64;
```
https://github.com/scriptex/svg64/blob/master/src/svg64.js#L120
https://github.com/scriptex/svg64/blob/master/src/svg64.js#L5
It's most likely an UMD module:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library